Src/ReleaseHistory.md

```diff
@@ -3,7 +3,8 @@
 ## Unreleased
 - BREAKING: Fix bug resulting in static validators returning `FailureLevel.Note` despite configured `FailureLevel`. [#645](https://github.com/microsoft/sarif-pattern-matcher/pull/645)
 - Update `sarif-sdk` submodule to commit [af69a1892954eb920c2a7e7b28e23effe35903c3](https://github.com/microsoft/sarif-sdk/commit/aaf69a1892954eb920c2a7e7b28e23effe35903c3), simply to keep current.
-- Update `spam` executable and dotnet library name to Sarif.PatternMatcher.Cli.
+- Update `spam` executable and dotnet library name to Sarif.PatternMatcher.Cli. 
+()- Update `Microsoft.Security.Utilities` to [v.1.4.0](https://github.com/microsoft/security-utilities/releases/tag/v1.4.0). [#662](https://github.com/microsoft/sarif-pattern-matcher/pull/662)
```

Comment:
>() [](http://example.com/codeflow?start=0&length=2)

can you remove this or add to the line above?

Reply:
Thanks. Not sure how that got in there.